api/schema/aws_account_resource.go

```diff
@@ -86,7 +89,7 @@ var (
 						},
 					},
 					attributeWithMode: attributeWithMode{
-						Mode: ImmutableAttributeMode,
+						Mode: WriteOnlyOnceAttributeMode,
```

Comment:
```suggestion
						Mode: ImmutableAttributeMode,
```

Reply:
same issue.

Reply:
role_external_id is a secret. writeonly once. we never expect it back from server.

Reply:
OK. Fair enough for this one.

Reply:
But if we want to treat this as a secret, it must be made `Sensitive: true`. Please do that.